docs/source/tutorial_chat_service.md

```diff
@@ -34,10 +34,8 @@ overworld: MessengerOverworld
 max_workers: 30
 opt:  # Additional model opts go here
   debug: True
-  model: legacy:seq2seq:0
-  model_file: models:convai2/seq2seq/convai2_self_seq2seq_model
-  override:
-    model: legacy:seq2seq:0
+  model: transformer/generator
```

Comment:
Totally ok if you don't want to do this, but for parity it'd be good to have the example config mirror the actual config you changed, i.e. to have a `models` key under `opt` with this config. e.g.:

```
opt:
  debug: True
  models:
    blender_90M:
      model: transformer/generator
      model_file: zoo:blender/blender_90M/model
```